server/services/userService.ts

```diff
@@ -1,19 +1,24 @@
 import logger from '../../logger'
-import type { CaseloadClient, HmppsAuthClient, RestClientBuilder, RestClientBuilderWithoutToken } from '../data'
+import type { CaseloadClient, HmppsManageUsersClient, RestClientBuilder } from '../data'
 import { StringUtils } from '../utils'
 import type { UserDetails } from '@accredited-programmes/users'
 import type { Caseload } from '@prison-api'
 
 export default class UserService {
   constructor(
-    private readonly hmppsAuthClientBuilder: RestClientBuilderWithoutToken<HmppsAuthClient>,
+    private readonly hmppsManageUsersClientBuilder: RestClientBuilder<HmppsManageUsersClient>,
     private readonly caseloadClientBuilder: RestClientBuilder<CaseloadClient>,
   ) {}
 
-  async getUser(token: Express.User['token']): Promise<UserDetails> {
-    const hmppsAuthClient = this.hmppsAuthClientBuilder()
-    const user = await hmppsAuthClient.getUser(token)
-    const caseloads = await this.getCaseloads(token)
+  async getUserWithDetails(token: Express.User['token']): Promise<UserDetails> {
```

Comment:
Wondering whether this should be called something else.  `getCurrentUserWithDetails` or just `getCurrentUser`?

Reply:
I think `getCurrentUserWithDetails` works well